### DISCUSSION TITLE
Vorteile von OpenDTU-OnBattery vs. IOBroker Script?

### DISCUSSION THREAD

**Discussion by Chris0706** [hoylabs/OpenDTU-OnBattery#1288] [General]
Hallo zusammen,

bin gerade auf dieses Projekt gestoßen und finde es prinzipiell sehr cool! Habe aktuell allerdings schon eine Nulleinspeiseregelung mit Akku im Betrieb über IOBroker, Shelly 3EM und OpenDTU/Growatt NOAH2000/HM-800 und frage mich nun, gibt es Vorteile, wenn ich die Regelung über OpenDTU-OnBattery machen würde? Mein Script arbeitet zum Beispiel nur im 5 Sekunden Intervall, weil es nicht schneller die Daten über MQTT bekommt.

Christian

**Comment by Gururom** [comment]
Ich habe im Moment ein Takt von 2 Sekunden eingestellt bei openDTUoB, läuft einwandfrei.
Ja der Vorteil ist, kein separater Rechner der laufen muss, ein Teil was expliziet nur zur Reglung da ist.
Zugriff idealer weise auf den WR (Hoymiles)direkt und auf den Shelly per JSON.
Kann irgendwo innerhalb des WLAN und WR Sendebereichs versteckt werden.
Also ich find es super.
Habe im Moment 2 WR einen HMS600 und einen HMS1600 4T angemeldet und lass den großen regeln
